app/routers/weekly_tasks.py

```diff
@@ -0,0 +1,194 @@
+import datetime
+
+from fastapi import APIRouter, Depends, Request, Form
+from starlette.responses import RedirectResponse
+from starlette.status import HTTP_302_FOUND
+
+from app.database.database import get_db
+from app.database.models import User, WeeklyTask
+from app.dependencies import templates
+from app.internal.weekly_tasks import (
+    remove_weekly_task, get_w_t_from_input,
+    create_weekly_task, change_weekly_task)
+
+
+router = APIRouter(
+    prefix="/weekly-tasks",
+    tags=["weekly-tasks"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+def get_placeholder_user():
+    user = User(
+        username='demo_user',
+        email='demo@email.po',
+        password='s3jsd183b13',
+        full_name='The Demo'
+    )
+    return user
+
+
+def get_user(demo_user, session):
+    user = session.query(User).filter_by(username=demo_user.username).first()
+    if not user:
+        session.add(demo_user)
+        session.commit()
+        user = session.query(User).filter_by(id=1).first()
+    return user
+
+
+def get_checked_days(days=""):
+    days_list = days.split(", ")
+    day_names = {
+        'Sun': 'Sunday',
+        'Mon': 'Monday',
+        'Tue': 'Tuesday',
+        'Wed': 'Wednesday',
+        'Thu': 'Thursday',
+        'Fri': 'Friday',
+        'Sat': 'Saturday'
+    }
+    checked_days = []
+    for day, day_full_name in day_names.items():
+        if day in days_list:
+            day = day.lower()
```

Comment:
It is common for both the if and the else - you should not put it in both of them